lib/order.rb

```diff
@@ -0,0 +1,53 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'yaml'
+
+class Order
+
+  def initialize
+    @dishes = []
+    @total = 0
+  end
+
+  def add(dish)
+    update_total(dish)
+    @dishes << dish
+  end
+
+  def review
+    "Your order:\n"+readable_dishes+"Total price: $"+@total.to_s
+  end
+
+  def dishes
+    @dishes.dup
+  end
+
+  def place(send_text = true)
+    if send_text
+      tokens = YAML::load(File.open('./lib/tokens.yml'))
+      account_sid = tokens["account_sid"]
+      auth_token = tokens["auth_token"]
+      client = Twilio::REST::Client.new account_sid, auth_token
+      client.account.messages.create(
+      :from => tokens["from_number"],
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)